pages/api/hakkasan.ts

```diff
@@ -0,0 +1,54 @@
+import { NextApiResponse } from 'next';
+import withCache, { conditionallyUpdateCache } from '../../lib/withCache';
+import artists from '../../data/artists.json';
+import events from '../../data/events.json';
+import venues from '../../data/venues.json';
+import type { HakkasanRequest } from '../../lib/withCache';
+
+const handler = async (
+  req: HakkasanRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  const {
+    method,
+    query: { ref },
+  } = req;
+  switch (method) {
+    case 'POST':
+      break;
+    case 'PUT':
+      break;
+    case 'PATCH':
+      break;
+    default: {
+      try {
+        if (process.env.NODE_ENV !== 'production') {
+          res.status(200).send({
```

Comment:
Might be nice to include a small timeout here to mimic some latency that would occur if we were to otherwise actually fetch from the API. That way we'd be able to have a more realistic flow from loading the data during development.